CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+# Contributing to RxJS Docs
+
+[Read and abide by the Code of Conduct](CODE_OF_CONDUCT.md)! Even if you don't read it,
+it still applies to you. Ignorance is not an exemption.
+
+Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Submitting a Pull Request (PR)](#submitting-a-pull-request-pr)
+  - [After your pull request is merged](#after-your-pull-request-is-merged)
+- [Commit Message Guidelines](#commit-message-guidelines)
+  - [Commit Message Format](#commit-message-format)
+  - [Revert](#revert)
+  - [Type](#type)
+  - [Scope](#scope)
+  - [Subject](#subject)
+  - [Body](#body)
+  - [Footer](#footer)
+
+  <!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+(This document is a work and progress and is subject to change)
+
+## Building/Testing
+
+The build and test structure is fairly primitive at the moment. There are various npm scripts that can be run:
+
+- start: runs the dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+- test: runs tests with `jasmine`, must have built prior to running.
+- build: build artifacts will be stored in the `dist/` directory
+- build-prod: builds for production using the `--prod` flag
+- commit: runs git commit wizard for passing rxjs-github-bot message validator
+
+## Submitting a Pull Request (PR)
+Before you submit your Pull Request (PR) consider the following guidelines:
+
+* Search [GitHub](https://github.com/ReactiveX/rxjs-docs/pulls) for an open or closed PR
+  that relates to your submission. You don't want to duplicate effort.
+* Make your changes in a new git branch:
+
+     ```shell
+     git checkout -b my-fix-branch master
+     ```
+
+* Create your patch, following [code style guidelines](#coding-style-guidelines), and **including appropriate test cases**.
+* Run the full test suite and ensure that all tests pass.
+* Run the micro and macro performance tests against your feature branch and compare against master
+  to ensure performance wasn't changed for the worse.
+* Commit your changes using a descriptive commit message that follows our
+  [commit message guidelines](#commit-message-guidelines). Adherence to these conventions
+  is necessary because release notes are automatically generated from these messages.
+
+     ```shell
+     git commit -a
```

Comment:
not for this PR, but probably we should have same tooling as rxjs repo to support `npm run commit` commit wizard.

Reply:
welp, actually this PR has it - let's add into guides as well (and recommend to use it)